### ISSUE TITLE
Color contrast issues on category descriptions

### ISSUE THREAD

**Issue by alexgibson** [mozilla/rise25#25]
Firefox dev tools a11y checker reports that some of the category descriptions fail colour contrast checks:

<img width="349" alt="image" src="https://github.com/mozilla/rise25/assets/400117/f860f981-e376-4345-b09a-9f2723aa024f">

![image](https://github.com/mozilla/rise25/assets/400117/9ad2570d-945d-4a15-8607-daf84ca83270)

It's probably worth checking over each section header for this issue.
